tools/binding-docu-generator/src/main/java/org/eclipse/smarthome/tools/docgenerator/models/State.java

```diff
@@ -0,0 +1,94 @@
+package org.eclipse.smarthome.tools.docgenerator.models;
+
+import org.eclipse.smarthome.tools.docgenerator.data.OptionList;
+
+import java.math.BigDecimal;
+
+public class State implements Model<org.eclipse.smarthome.tools.docgenerator.schemas.State> {
```

Comment:
This class needs some protection - if there is no state set for a channel, then it will NPE in multiple places. Basically, add ```if(delegate == null) return XX;``` needs to be added, or maybe there's a better option to initialise a good default.